fe/fe-core/src/main/java/com/starrocks/catalog/HudiTable.java

```diff
@@ -94,16 +95,20 @@ public enum HudiTableType {
         COW, MOR, UNKNOWN
     }
 
-    private String resourceName;
     private String catalogName;
+    @SerializedName(value = "dn")
     private String hiveDbName;
+    @SerializedName(value = "tn")
     private String hiveTableName;
-
+    @SerializedName(value = "rn")
+    private String resourceName;
+    @SerializedName(value = "pcn")
     private List<String> partColumnNames = Lists.newArrayList();
     // dataColumnNames stores all the non-partition columns of the hudi table,
     // consistent with the order defined in the hudi table
+    @SerializedName(value = "dcn")
     private List<String> dataColumnNames = Lists.newArrayList();
-
+    @SerializedName(value = "prop")
```

Comment:
need to register Table and all its subclasses?

Reply:
done